x-pack/filebeat/module/cisco/amp/test/cisco_amp2.ndjson.log-expected.json

```diff
@@ -0,0 +1,5676 @@
+            "user@testdomain.com"
+        ],
+        "service.type": "cisco",
+        "tags": [
+            "cisco-amp",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2021-01-15T10:27:38.000Z",
+        "cisco.amp.computer.active": true,
+        "cisco.amp.computer.connector_guid": "test_connector_guid",
+        "cisco.amp.computer.external_ip": "8.8.8.8",
+        "cisco.amp.computer.network_addresses": [
+            {
+                "ip": "10.10.10.10",
+                "mac": "63:5f:47:2b:89:91"
+            }
+        ],
+        "cisco.amp.connector_guid": "test_connector_guid",
+        "cisco.amp.detection": "W32.File.MalParent",
+        "cisco.amp.detection_id": "6533671380737589308",
+        "cisco.amp.event_type_id": 1090519054,
+        "cisco.amp.file.disposition": "Malicious",
+        "cisco.amp.group_guids": [
+            "test_group_guid"
+        ],
+        "cisco.amp.related.mac": [
+            "63:5f:47:2b:89:91"
+        ],
+        "cisco.amp.timestamp_nanoseconds": 605000000,
+        "event.action": "Threat Detected",
+        "event.dataset": "cisco.amp",
+        "event.id": 6533671380737589000,
+        "event.kind": "alert",
+        "event.module": "cisco",
+        "event.severity": 2,
+        "file.hash.md5": "b99e0a8c56f963246b6464b9fffbf7a2",
+        "file.hash.sha1": "b024546a49bad1bd60fccef0a5d11b55f9a442c4",
+        "file.hash.sha256": "b1380fd95bc5c0729738dcda2696aa0a7c6ee97a93d992931ce717a0df523967",
+        "file.name": "ekjrngjker.exe",
+        "file.path": "C:\\ekjrngjker.exe",
+        "fileset.name": "amp",
+        "host.hostname": "Demo_AMP_Threat_Audit",
+        "host.name": "Demo_AMP_Threat_Audit",
+        "host.user.name": "user@testdomain.com",
+        "input.type": "log",
+        "log.offset": 30074,
+        "related.hash": [
+            "b1380fd95bc5c0729738dcda2696aa0a7c6ee97a93d992931ce717a0df523967",
+            "b99e0a8c56f963246b6464b9fffbf7a2",
+            "b024546a49bad1bd60fccef0a5d11b55f9a442c4"
+        ],
+        "related.ip": [
+            "10.10.10.10"
+        ],
+        "related.user": [
+            "user@testdomain.com"
+        ],
+        "service.type": "cisco",
+        "tags": [
+            "cisco-amp",
+            "forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2021-01-15T10:26:38.000Z",
+        "cisco.amp.computer.active": true,
+        "cisco.amp.computer.connector_guid": "test_connector_guid",
+        "cisco.amp.computer.external_ip": "8.8.8.8",
+        "cisco.amp.computer.network_addresses": [
+            {
+                "ip": "10.10.10.10",
+                "mac": "63:5f:47:2b:89:91"
+            }
+        ],
+        "cisco.amp.connector_guid": "test_connector_guid",
+        "cisco.amp.detection": "W32.File.MalParent",
+        "cisco.amp.detection_id": "6533671123039551547",
+        "cisco.amp.event_type_id": 1090519054,
+        "cisco.amp.file.disposition": "Malicious",
```

Comment:
could we map this to `event.category = "malware"`?

Reply:
Seems like I want to go through the source data again and see if I can add some proper event.category and event.type in here, as its mostly just event.kind at this moment.